src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -195,15 +203,34 @@ ov::frontend::InputModel::Ptr FrontEnd::load_impl(const std::vector<ov::Any>& va
 std::shared_ptr<ov::Model> FrontEnd::convert(const ov::frontend::InputModel::Ptr& model) const {
     auto f = convert_partially(model);
 
-    auto unsupported_operations = get_unconverted_types_from_model(f);
+    std::unordered_map<std::string, std::string> failures;
+    std::vector<std::string> unsupported_operations;
+    get_unsupported_operations_and_failures(f, unsupported_operations, failures);
+
+    std::string exception_message;
```

Comment:
string += will reallocate buffer (like vector) 
stringstring doesn't reallocate because contains linked buffers or something like that.
the string concatenation will be faster with stringstream

Reply:
Fixed.